Cubical/HITs/Hopf.agda

```diff
@@ -320,3 +322,20 @@ JoinS¹S¹≡TotalHopf = isoToPath (iso JoinS¹S¹→TotalHopf
 
 S³≡TotalHopf : S³ ≡ TotalHopf
 S³≡TotalHopf = S³≡joinS¹S¹ ∙ JoinS¹S¹≡TotalHopf
+
+open Iso renaming (inv to inv')
```

Comment:
What is the other use on `inv` in scope? Maybe that should be renamed?

Reply:
That's the function from S1 to S1 sending loop to sym loop. I renamed it to invLooper